Codeinwp/otter-blocks
# Add the option to hide the blocks in Inserter.

Closes #.


### Summary


Add UI for disabling Otter Blocks in Editor Inserter. This feature allows users to select what blocks they want to use.

The blocks on the page are not affected.

ℹ️ Added a shortcut in named `Blocks`s below Settings. When you enter, it will create a redirect to Dashboard with the Tab set on `Blocks`

### Screenshots 

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/43a96fa6-e64c-4355-8f04-5f32275efd9d)

https://github.com/Codeinwp/otter-blocks/assets/17597852/a066b765-342d-4de7-af58-4bb812032114

----

### Test instructions


Test the enable/disable functions. They should work like in the video attached. 



---- 

### Checklist before the final review
